tests/typecheck/db/models/test_constraints.yml

```diff
@@ -35,3 +35,13 @@
         main:4: note: Possible overload variants:
         main:4: note:     .*
         main:4: note:     .*
+
+-   case: constraint_violation_error_message
```

Comment:
We don't need tests for simple things. We only test plugins / complex overloads / etc

Reply:
Agree